diff --git a/Core/Frameworks/Baikal/Core/Server.php b/Core/Frameworks/Baikal/Core/Server.php
index b4765d1..140f7b1 100644
--- a/Core/Frameworks/Baikal/Core/Server.php
+++ b/Core/Frameworks/Baikal/Core/Server.php
@@ -133,6 +133,8 @@ class Server {
 
         if ($this->authType === 'Basic') {
             $authBackend = new \Baikal\Core\PDOBasicAuth($this->pdo, $this->authRealm);
+        } elseif ($this->authType === 'LDAP-UserBind') {
+            $authBackend = new \Baikal\Core\LDAPUserBindAuth($this->pdo, $this->authRealm);
         } elseif ($this->authType === 'Apache') {
             $authBackend = new \Sabre\DAV\Auth\Backend\Apache();
         } else {
diff --git a/Core/Frameworks/Baikal/Model/Config/Standard.php b/Core/Frameworks/Baikal/Model/Config/Standard.php
index 416827f..2f86d84 100644
--- a/Core/Frameworks/Baikal/Model/Config/Standard.php
+++ b/Core/Frameworks/Baikal/Model/Config/Standard.php
@@ -42,7 +42,11 @@ class Standard extends \Baikal\Model\Config {
         // While not editable as will change admin & any existing user passwords,
         // could be set to different value when migrating from legacy config
         "auth_realm"            => "BaikalDAV",
-        "base_uri"              => ""
+        "base_uri"              => "",
+        "dav_ldap_uri"              => "ldapi:///",
+        "dav_ldap_dn_template"      => "uid=%n,dc=example,dc=com",
+        "dav_ldap_displayname_attr" => "cn",
+        "dav_ldap_email_attr"       => "mail",
     ];
 
     function __construct() {
@@ -79,7 +83,30 @@ class Standard extends \Baikal\Model\Config {
         $oMorpho->add(new \Formal\Element\Listbox([
             "prop"    => "dav_auth_type",
             "label"   => "WebDAV authentication type",
-            "options" => ["Digest", "Basic", "Apache"]
+            "options" => ["Digest", "Basic", "Apache", "LDAP-UserBind"]
+        ]));
+        $oMorpho->add(new \Formal\Element\Text([
+            "prop"  => "dav_ldap_uri",
+            "label" => "LDAP URI"
+        ]));
+
+        $oMorpho->add(new \Formal\Element\Text([
+            "prop"    => "dav_ldap_dn_template",
+            "label"   => "LDAP DN template",
+            "popover" => [
+                "title"   => "posible placeholder",
+                "content" => "<strong>%n</strong> - username<br /><strong>%u</strong> - user part of username , when it is an email address)<br /><strong>%d</strong> - domain part",
+            ]
+        ]));
+
+        $oMorpho->add(new \Formal\Element\Text([
+            "prop"  => "dav_ldap_displayname_attr",
+            "label" => "LDAP attribute for DisplayName"
+        ]));
+
+        $oMorpho->add(new \Formal\Element\Text([
+            "prop"  => "dav_ldap_email_attr",
+            "label" => "LDAP attribute for eMail"
         ]));
 
         $oMorpho->add(new \Formal\Element\Password([
